lib/rspec/mocks/minitest_integration.rb

```diff
@@ -0,0 +1,15 @@
+require 'rspec/mocks'
+
+Minitest::Expectation.class_eval do
```

Comment:
One other thought: it can often work better to define our monkey patch methods in a module we own, and mix that in to `Minitest::Expectation` rather than re-opening `Minitest::Expectation` directly.  Among other things, it means that `Minitest::Expectation.instance_method(:to).owner` would list itself as being defined by an rspec module rather than a minitest one.  See http://www.justinweiss.com/blog/2015/01/20/3-ways-to-monkey-patch-without-making-a-mess/ for more discussion of this technique.